local_pathfinding/local_path.py

```diff
@@ -7,13 +7,33 @@
 
 
 class LocalPathState:
+    """Gathers and stores the state of the Sailbot.
+
+    Attributes:
+        `postion` (GPS): The latitude (float32) and longitudinal (float32) coordinates of the Sailbot.
+        `speed` (GPS): The speed (float32) of the Sailbot at that position. Units: km/hr.
+        `heading` (GPS): The direction (float32) in which the Sailbot is Sailing at. Units: Degrees.
+        `ais_ships` (AISShips): List of ships.
+        `global_path` (GlobalPath): Objects of all the global way points which the Sailbot will travel to.
+        `wind_speed` (WindSensor): The wind speed. Units: km/hr.
+        `wind_direction` (WindSensor): The wind direction (int16) towards the boat. Units: Degrees.
+    """
+
     def __init__(
         self,
         gps: GPS,
         ais_ships: AISShips,
         global_path: GlobalPath,
         filtered_wind_sensor: WindSensor,
     ):
+        """Initializes the local path state.
+
+        Args:
+            `gps` (GPS): Data from the GPS sensors.
+            `ais_ships` (AISShips): Data from the AIS receiver.
+            `global_path` (GlobalPath): Data from the Globalpath server.
+            `filtered_wind_sensor` (WindSensor): Data from the windsensors.
```

Comment:
I would just say `The xxx msg.`

Reply:
I have said the following:
            `gps`: The GPS msg.
            `ais_ships`: The AISShips msg.
            `global_path`: The GlobalPath msg.
            `filtered_wind_sensor`: The WindSensor msg.